### PR TITLE
Update refresh ig 2

### PR DESCRIPTION
**Pull request by ddieppois**
**Description**

- Github Issue:  
- [ ] I've read the contribution guidelines 
- [ ] Code compiles without errors
- [ ] Tests are created / updated
- [ ] Documentation is created / updated

By creating this PR you acknowledge that your contribution will be licensed under Apache 2.0

### CODE REVIEW

File: tooling/src/main/java/org/opencds/cqf/tooling/cql_generation/builder/VmrToModelElmBuilder.java [hunk 1]
```diff
@@ -845,7 +847,7 @@ public BigDecimal parseDecimal(String value) {
             BigDecimal result = (BigDecimal)decimalFormat.parse(value);
             return result;
         } catch (ParseException e) {
-            throw new IllegalArgumentException(String.format("Could not parse number literal: %s", value), e);
+            throw new IllegalArgumentException(String.format("Could not parse number literal: %s %s", value, e));
```
Comment (JPercival) [inline tooling/src/main/java/org/opencds/cqf/tooling/cql_generation/builder/VmrToModelElmBuilder.java:850]: issue: Looks like the original code was right to me.

File: tooling/src/test/java/org/opencds/cqf/tooling/processor/IGProcessorTest.java [hunk 2]
```diff
@@ -30,11 +33,23 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import com.google.gson.Gson;
+import java.io.*;
+import java.nio.file.DirectoryStream;
```
Comment (JPercival) [inline tooling/src/test/java/org/opencds/cqf/tooling/processor/IGProcessorTest.java:37]: question: Are all these extra imports needed?

File: tooling/src/main/java/org/opencds/cqf/tooling/processor/BaseProcessor.java [hunk 3]
```diff
@@ -122,7 +122,7 @@ public void initializeFromIg(String rootDir, String igPath, String fhirVersion)
         this.fhirVersion = sourceIg.getFhirVersion().get(0).getCode();
         packageId = sourceIg.getPackageId();
         canonicalBase = determineCanonical(sourceIg.getUrl());
-        packageManager = new NpmPackageManager(sourceIg, this.fhirVersion);
+        packageManager = new NpmPackageManager(sourceIg, "4.3.0");
```
Comment (JPercival) [inline tooling/src/main/java/org/opencds/cqf/tooling/processor/BaseProcessor.java:125]: question: Do we really want to pin to "4.3.0" here?

**Review by JPercival — APPROVED** [review]

**Review by c-schuler — APPROVED** [review]
Looks good!

**Pull request merged**